src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,49 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer ferrari = new Manufacturer(4L, "ferrari", "Italy");
```

Comment:
It's better to separate logically grouped lines of code by empty blocks for the sake of readability and easier understanding in the future either by you or by other devs.